pyproject.toml

```diff
@@ -92,7 +92,7 @@ mypy = [
 
 [tool.black]
 line-length = 99
-target-version = ["py38", "py39", "py310", "py311", "py312"]
+target-version = ["py39", "py310", "py311", "py312", "py313"]
```

Comment:
Explicitly keep this setting to allow for easier updates when dropping a python version. Without it, the formatting change would need to happen in the same PR.